### PR TITLE
 README.rst: update report image and GIF link

### PR DESCRIPTION
**Pull request by Lavenderjiang**
The new GIF and report image make README more up to date.

 Closes https://github.com/coala/coala-html/issues/141

 new file:   coalahtml/_coalahtml/.coafile

 This .coafile allows coala-html to successfully run in coalahtml/_coalahtml directory. I was able to make a successful demo of coala-html with the addition of this file.

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: coalahtml/_coalahtml/.coafile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+[js]
```
Comment (jayvdb) [inline coalahtml/_coalahtml/.coafile]: please remove this file.  It is already covered by the file https://github.com/coala/coala-html/blob/master/.coafile .
I do appreciate that you needed it to generate the image, and so it was helpful for that purpose.  but it isnt needed in the repository

Comment (jayvdb) [reply coalahtml/_coalahtml/.coafile]: And likewise remove everything in the commit message after `Closes #141`.

**Comment by hemangsk** [comment]
@Lavenderjiang Please address the comments by reviewers and let us know in case you have any doubt / query  :)

**Comment by jayvdb** [comment]
Close due to inactivity.

**Pull request closed without merging**